### PR TITLE
Fixes for RelayedV3 and Cookbook update

### PR DESCRIPTION
**Pull request by popenta**

### CODE REVIEW

File: examples/Cookbook.ipynb [hunk 1]
```diff
@@ -479,6 +480,126 @@
     "print(\"Transaction data:\", transaction.data.decode())"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Additionally, we also have a method that combines the above methods and is able to identify the kind of transfer that we intend to perform based on it's parameters. The method can be used as follows:\n",
```
Comment (andreibancioiu) [inline examples/Cookbook.ipynb:487]: :+1:

File: examples/Cookbook.ipynb [hunk 2]
```diff
@@ -479,6 +480,126 @@
...
+    "\n",
+    "print(\"Transaction:\", transaction_converter.transaction_to_dictionary(transaction))\n",
+    "print(\"Transaction data:\", transaction.data.decode())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Alternatively, the protocol now supports sending both native and esdt tokens in the same transaction. If a `native_amount` is provided together with `token_transfers`, the native token will also be included in the `MultiESDTNFTTrasfer` built-in function call."
```
Comment (andreibancioiu) [inline examples/Cookbook.ipynb:568]: "now supports" is somehow incorrect. It will support. If there is an already published page on docs, we can link to it. Otherwise, let's use the future tense.

File: examples/Cookbook.ipynb [hunk 3]
```diff
@@ -654,6 +789,70 @@
     "print(transaction_converter.transaction_to_dictionary(relayed_tx))"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Relayed V3\n",
+    "\n",
+    "This is the latest iteration of relayed transactions and we highly recommend using this version over the previous ones."
```
Comment (andreibancioiu) [inline examples/Cookbook.ipynb:798]: Add a link to the official docs. And maybe rephrase to say "we recommend using this version ... as soon as it becomes available on the Network" (or something similar).

File: examples/Cookbook.ipynb (hunk 2 above)
Comment (popenta) [reply examples/Cookbook.ipynb:568]: As far as I've seen, there is no docs page yet, but I've changed the text to be:
```
the protocol will support (in the very near future) sending ...
```

File: examples/Cookbook.ipynb (hunk 3 above)
Comment (popenta) [reply examples/Cookbook.ipynb:798]: changed the message and will add a link to the docs when that is available.

**Review by andreibancioiu — APPROVED** [review]
